morphia/src/main/java/org/mongodb/morphia/DatastoreImpl.java

```diff
@@ -250,6 +260,40 @@ public void ensureCaps() {
     }
 
     @Override
+    public void enableDocumentValidation() {
+        for (final MappedClass mc : mapper.getMappedClasses()) {
+            Validation validation = (Validation) mc.getAnnotation(Validation.class);
+            if (validation != null) {
+                String collectionName = mc.getCollectionName();
+                CommandResult result = getDB()
+                    .command(new BasicDBObject("collMod", collectionName)
+                                 .append("validator", Document.parse(validation.value()))
+                                 .append("validationLevel", validation.level().getValue())
+                                 .append("validationAction", validation.action().getValue())
+                            );
+
+                if (!result.ok()) {
```

Comment:
Question: If there was an existing validation - would it be overwritten? Or would it throw an error?